postgis/Dockerfile

```diff
@@ -16,17 +16,18 @@ RUN apt-get update \
     proj-bin \
     proj-data \
     # - JSON
-    libjson0 \
-    libjson0-dev \
+    libjson-c3 \
+    libjson-c-dev \
     python-simplejson \
     # - GEOS
-    libgeos-c1 \
+    libgeos-c1v5 \
     libgeos-dev \
     # - GDAL
+    libgdal20 \
     gdal-bin \
-    libgdal1-dev \
+    libgdal-dev \
     # PostGIS
-    liblwgeom-2.1.4 \
+    liblwgeom-2.3\* \
```

Comment:
Yep. I wild carded because that package looked a bit volatile in naming.  Typically, you see the Debian packages named `liblwgeom` and the version being separate, so my approach when you have a package with "proper" naming, I would have specified the version I want in an apt-get line, like this:

```
apt-get install liblwgeom=2.3* docker-ce=17.04* -y
```
Which would get you the latest of those major versions.

In the case of liblwgeom, it appears they packaged it with a name of liblwgeom-2.3.x and then also have a liblwgeom-2.4.x -- but the major version of this package seems to have been packaged in a way that concatenated the version into the package name.  Honestly, this was a quick fix based on what I found in an `apt search liblwgeom` output.  Figured it'd be more stable long-term.

Reply:
Wouldn't it install _every_ package matching the pattern. [Manpage](https://manpages.debian.org/stretch/apt/apt-get.8.en.html) for apt-get in Debian Stretch says:
> If no package matches the given expression and the expression contains one of '.', '?' or '\*' then it is assumed to be a POSIX regular expression, and it is applied to all package names in the database. Any matches are then installed (or removed). Note that matching is done by substring so 'lo.*' matches 'how-lo' and 'lowest'. If this is undesired, anchor the regular expression with a '^' or '$' character, or create a more specific regular expression.

Reply:
Or does escaping the asterisk have a different effect? I was a little confused about that part.

Reply:
The escaping just ensures that a literal `*` character is sent to the apt command as an argument.  You probably don't need to do this, if you are more selective about the specific package that you wanted to install.  I was in somewhat of a hurry when I pieced this together, and thought it was an acceptable shortcut.  I'm totally ok with you taking anything that's helpful in this PR, harvesting it, and closing the PR.

Reply:
Ah, I see. Sorry I haven't merged this yet; all the containers build now, but I'm no longer able to log into the CartoDB editor application. I'm pretty sure the problem lies in the editor container, not in this one, but I'd like to get things back into a working state before I merge.

Reply:
Are you guys facing the same error as me?

```
rake aborted!
PG::Error: ERROR:  invalid option "extensions"
HINT:  Valid options in this context are: service, passfile, connect_timeout, dbname, host, hostaddr, port, application_name, keepalives, keepalives_idle, keepalives_interval, keepalives_count, sslmode, sslcompression, sslcert, sslkey, sslrootcert, sslcrl, requirepeer, krbsrvname, target_session_attrs, use_remote_estimate, fdw_startup_cost, fdw_tuple_cost, updatable
CONTEXT:  SQL statement "ALTER SERVER aggregation OPTIONS (ADD extensions 'postgis')"
PL/pgSQL function _cdb_setup_fdw(text,json) line 23 at EXECUTE
SQL statement "SELECT cartodb._CDB_Setup_FDW($1, $2)"
PL/pgSQL function _cdb_setup_fdw(text) line 6 at EXECUTE
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `exec'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `block in execute_query'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-rails-0.9.15/lib/sequel_rails/sequel/database/active_support_notification.rb:17:in `block in log_connection_yield'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/notifications.rb:123:in `block in instrument'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/notifications.rb:123:in `instrument'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-rails-0.9.15/lib/sequel_rails/sequel/database/active_support_notification.rb:11:in `log_connection_yield'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-rails-0.9.15/lib/sequel_rails/sequel/database/active_support_notification.rb:28:in `log_yield'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `execute_query'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `block in execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:111:in `check_disconnect_errors'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:411:in `_execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block (2 levels) in execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:423:in `check_database_errors'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block in execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:91:in `hold'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:79:in `execute_dui'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:72:in `execute_ddl'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:135:in `run'
/cartodb/app/models/user/db_service.rb:1256:in `block (2 levels) in connect_to_aggregation_tables'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:323:in `_transaction'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:285:in `block in transaction'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:278:in `transaction'
/cartodb/app/models/user/db_service.rb:1254:in `block in connect_to_aggregation_tables'
/cartodb/app/models/user.rb:717:in `in_database'
/cartodb/app/models/user/db_service.rb:1253:in `connect_to_aggregation_tables'
/cartodb/app/models/user.rb:1950:in `setup_aggregation_tables'
/cartodb/app/models/user.rb:1615:in `setup_user'
/cartodb/app/models/user.rb:350:in `after_create'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1585:in `block (2 levels) in _save'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_create'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1578:in `block in _save'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_save'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1573:in `_save'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block (2 levels) in save'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `block in checked_transaction'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:323:in `_transaction'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:285:in `block in transaction'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/database/query.rb:278:in `transaction'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `checked_transaction'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block in save'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1713:in `checked_save_failure'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `save'
/cartodb/lib/tasks/setup.rake:86:in `block (3 levels) in <top (required)>'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `call'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `block in execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `each'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `execute'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/task.rb:158:in `block in invoke_with_call_chain'
/opt/rubies/ruby-2.2.3/lib/ruby/2.2.0/monitor.rb:211:in `mon_synchronize'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/task.rb:151:in `invoke_with_call_chain'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/task.rb:144:in `invoke'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:116:in `invoke_task'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block (2 levels) in top_level'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `each'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block in top_level'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:88:in `top_level'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:66:in `block in run'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/lib/rake/application.rb:63:in `run'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/rake-0.9.2.2/bin/rake:33:in `<top (required)>'
/opt/rubies/ruby-2.2.3/bin/rake:23:in `load'
/opt/rubies/ruby-2.2.3/bin/rake:23:in `<top (required)>'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/cli/exec.rb:74:in `load'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/cli/exec.rb:28:in `run'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/cli.rb:424:in `exec'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/cli.rb:27:in `dispatch'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/cli.rb:18:in `start'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/exe/bundle:30:in `block in <top (required)>'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
/opt/rubies/ruby-2.2.3/lib/ruby/gems/2.2.0/gems/bundler-1.16.2/exe/bundle:22:in `<top (required)>'
/opt/rubies/ruby-2.2.3/bin/bundle:23:in `load'
/opt/rubies/ruby-2.2.3/bin/bundle:23:in `<main>'
Tasks: TOP => cartodb:db:create_dev_user
```